conf/api.profile.conf.js

```diff
@@ -0,0 +1,12 @@
+var profile = require('./profile.conf').config;
+var integrationProfile = require('./integration.profile.conf').config;
+
+integrationProfile.matchers = [
```

Comment:
you can also add those to 'profile' and so avoid the import of integration.profile.conf